README.md

```diff
@@ -0,0 +1,13 @@
+# About
+
+After applying changes to a qgisproject (styling, adding, ect...),
+ the next OGC request will trigger the reload of the project.
+ Thus applying the changes without having to restart the QGIS Server.
+
+Both projects stored on the server and in the database are reloaded.
```

Comment:
can the database part also be tested easily ?

Reply:
Both projects stored on the *file system* and in a database are reloaded.